docs/proposals/Disclosure/README.md

```diff
@@ -0,0 +1,111 @@
+  assistance
+- Include `aria-expanded` attribute on the trigger to communicate to assistive
+  technology
+- Include `aria-controls` attribute ties on the trigger to the content it
+  controls using the `id` of the collapsible container
+- The trigger contains a downward-pointing-arrow to hint that it can be
+  expanded. when the disclosure item is in expanded state, this is rotated 180
+  degrees to point upwards
+- The icon will be given an `aria-hidden=”true”` attribute to hide it from
+  assistive technologies, as well as `focusable=”false”` to address an
+  inconsistency in IE and older versions of Edge
+
+## Responsiveness
+
+- This component should handle both click and touch, as well as keyboard inputs
+- This component should fill the width of its container
+- If the component is less than `375px` wide, the title will wrap and should not
+  be truncated
+- Text increases/decreases when user zooms in/out
+
+## Mockup
+
+!["Disclosure Mockup"](https://gist.githubusercontent.com/kingstonfung/b882aa211f4213b1ac89bd515dc48a50/raw/efef596279fa0070d26b67cd364e533ca3d572bd/image2.png)
+
+## Interface
+
+```jsx
+<Disclosure title="Advanced Options">
+  <Content>Example content here</Content>
+</Disclosure>
+```
+
+## Props Table
+
+| name            | type         | default   | description                                                                                                                                                                                                             |
+| --------------- | ------------ | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| title           | string       | -         | Title for the disclosure pane                                                                                                                                                                                           |
+| open            | boolean      | undefined | Control if the disclosure component is open or closed                                                                                                                                                                   |
+| defaultOpen     | boolean      | false     | This sets the default open state of the disclosure. By default the disclosure is closed. For use when the component is being used as an [Uncontrolled](https://reactjs.org/docs/uncontrolled-components.html) component |
+| onRequestToggle | (() => void) | void      | This function would toggle the open state. For use when the component is being used as a [Controlled](https://reactjs.org/docs/forms.html#controlled-components) component                                              |
+
+## Notes
+
+**Trade-offs, considerations, and things we want to remember**
+
+- `onchange` event for tracking when opens
```

Comment:
@kingstonfung / @seanhealy do you recall what this point is referring to?

Reply:
@chris-at-jobber : The mention here I believe was to set out a reminder that we _may_ , in the future, setup a handler that receives expanded/collapsed events. For example so other parts of a page, SPA or not, can subscribe to this and react accordingly. Hope this makes sense. Thanks!

Reply:
Makes sense! Let's include that detail here:

```suggestion
- We may, in the future, setup an `onchange` handler that receives expanded/collapsed events. For example so other parts of a page, SPA or not, can subscribe to this and react accordingly.
```